### PR TITLE
feat(streaming): support streaming logs by pipeline mode

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
support streaming logs by pipeline mode

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=370858&iterationID=12783&tab=TASK&type=TASK)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |   support streaming logs by pipeline mode           |
| 🇨🇳 中文    |   实现streaming消费日志用pipeline的模式           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: cmd/monitor/streaming/bootstrap.yaml [hunk 1]
```diff
@@ -189,20 +189,6 @@ log-storage-elasticsearch:
...
-  _enable: ${WRITE_LOG_TO_ES_ENABLE|WRITE_LOG_TO_CLICKHOUSE_ENABLE:true}
-  input:
-    topics: "${LOG_TOPICS:spot-container-log,spot-job-log}"
-    group: "${LOG_GROUP_ID:erda-logs-dev}"
-    offsets:
-      initial: earliest
-  id_keys: "${LOG_ID_KEYS:TERMINUS_DEFINE_TAG,terminus_define_tag,MESOS_TASK_ID,mesos_task_id}"
-  read_timeout: "5s"
-  buffer_size: ${LOG_BATCH_SIZE:4096}
-  parallelism: ${LOG_PERSIST_PARALLELISM:6}
```
Comment (sfwn) [inline cmd/monitor/streaming/bootstrap.yaml:202]: Where to set parallelism in pipeline mode, is `EXPORTER_CH_LOG_CURRENCY_NUM`?

File: internal/tools/monitor/oap/collector/lib/protoparser/spotlog/spotlog.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+	// setup level
+	if level, ok := data.Tags["level"]; ok {
+		data.Tags["level"] = strings.ToUpper(level)
+	} else {
+		data.Tags["level"] = "INFO"
+	}
+
+	// setup request id
+	if reqID, ok := data.Tags["request_id"]; ok {
```
Comment (sfwn) [inline internal/tools/monitor/oap/collector/lib/protoparser/spotlog/spotlog.go:103]: use const for literal strings.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: cmd/monitor/streaming/bootstrap.yaml (hunk 1 above)
Comment (chengjoey) [reply cmd/monitor/streaming/bootstrap.yaml:202]: that's it

File: internal/tools/monitor/oap/collector/lib/protoparser/spotlog/spotlog.go (hunk 2 above)
Comment (chengjoey) [reply internal/tools/monitor/oap/collector/lib/protoparser/spotlog/spotlog.go]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**